manim/logger.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import configparser
+
+from rich.logging import RichHandler
+from rich.console import Console
+from rich.theme import Theme
+#from .config import config
+
+#print(config)
+def parseTheme(fp):
+    config = configparser.ConfigParser()
+    config.read(fp)
+    theme=dict(config['log.color'])
+    customTheme=Theme(theme)
+    return customTheme
+try:
+    customTheme =  parseTheme("manim.cfg")
```

Comment:
This is related to the comment immediately above this one.

`manim.cfg` in the current working directory won't be the only configuration file, as far as I can tell.

Either you can also read the themes present in all the other cascading `manim.cfg` files that eulertour mentioned well be implementing [here](https://github.com/ManimCommunity/manim/pull/98#issuecomment-653929650), or you can the config-parser in config.py to do any parsing.